index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
```

Comment:
- [ ] **Required** Let's add the `CSS File link` here so CSS code will reflect in HTML file.

Reply:
I was not able to add a description to the pull request. Is that okay?